components/eamxx/cime_config/testdefs/testmods_dirs/eamxx/prod/yaml_outs/eamxx_output.decadal.4hourlyAVG_coarse.yaml

```
@@ -1,9 +1,9 @@
 %YAML 1.1
 ---
-filename_prefix: eamxx_output.decadal.6hourlyAVG_coarse.h
+filename_prefix: eamxx_output.decadal.4hourlyAVG_coarse.h
 iotype: pnetcdf
 averaging_type: average
-max_snapshots_per_file: 4  # one file per day
+max_snapshots_per_file: 1  # one file per day
```

This works for all the yaml files: maybe we should be more explicit, and instead request
```suggestion
storage_type: one_day
```

This feature has not been used (that I know of) in actual CIME cases, so we may hit some bumps regarding how CIME detects files to compare (via regex). I _think_ we fixed the regexes to accommodate this, but I'd have to check.

---

Also, with 4hourly output you need 6 snaps/file, not 1. Rather than correct the number every time, the `one_day` storage type may be more "permanent"...

---

Since the time horizon of the run is governed by the test specifiers (such as Ld1 and the likes), at this point I would not specify any storage type or max snaps/file. If you don't specify anything, eamxx will stuff all slices in the same file, regardless.

---

Yeah

---

> This was actually the problem with the test in the nightlies. The run only goes for 4 hours for some reason, thus missing out all the output. I want to make all output very frequent so that it is all caught in the ERS comp stuff

I don't understand, this seems like motivation for moving to high frequency output, but not changing the max snapshots per file. And for longer simulations, we definitely have N>1 snapshots per file. Also, note that if we move everything to high frequency (e.g., 1 hour), then we won't be exercising the case where the output frequency is less than the call frequency (e.g., 1 hour timestep, 3-hourly average output, or something like that).